app/services/bulk_upload/sales/year2023/row_parser.rb

```diff
@@ -425,6 +425,14 @@ class BulkUpload::Sales::Year2023::RowParser
             },
             on: :before_log
 
+  validates :field_105,
+            inclusion: {
+              in: [1, 2],
+              if: proc { field_89.present? && field_89 != 100 && shared_ownership? },
```

Comment:
I can see this matches the ACs but I'm a bit confused because they don't seem to match the text description, which suggests "don't know" should never be allowed for 2023. Do you know what's going on there?

Reply:
Ah yes, so, the ACs are correct, they got changed at some point, there's more context on the last comment

Reply:
Ah yes, so, the ACs are correct, they got changed at some point there's more context on the last comment